helm-charts/nginx-ingress/templates/post-upgrade-job.yaml

```diff
@@ -18,6 +18,10 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/name: {{ .Release.Name }}
     helm.sh/chart: {{ .Chart.Name }}
+  annotations:
+    "helm.sh/hook": post-upgrade
```

Comment:
this hook should also be triggered in post-install case.

Reply:
Done.